apps/balancer-tools/src/app/apr/(components)/HeaderEndButton.tsx

```diff
@@ -61,24 +63,31 @@ export default function HeaderEndButton() {
         placeholder="Select a round"
         value={selectedRound}
         onValueChange={(value) => {
+          setSelectedRound(value);
           router.push(
             !poolId
               ? `/apr/round/${value}`
               : `/apr/pool/${network}/${poolId}/round/${value}`,
           );
         }}
-        className=""
       >
         {typeof roundId === "undefined" ? (
           <SelectItem value="">Select a round</SelectItem>
         ) : (
           ""
         )}
-        {ALL_ROUNDS.map((round) => (
-          <SelectItem key={round.value} value={round.value}>
-            {round.label}
-          </SelectItem>
-        ))}
+        <div className="flex flex-col gap-y-1">
+          {ALL_ROUNDS.map((round) => (
+            <SelectItem key={round.value} value={round.value}>
+              <div className="flex gap-x-2 items-center">
+                <Badge color="darkBlue" size="sm">
+                  Round {round.value}
+                </Badge>
```

Comment:
this ended up going out of the Select boundary on mobile, see print. Think it's easier to fix start working on it now you already have the context of this than to come back to all of it later on.

Reply:
![image](https://github.com/bleu-studio/balancer-tools/assets/32641844/9dd81dd6-f070-4c67-8f25-2bc7cfe97c6b)